copilot/rails/manifest.yml

```diff
@@ -48,35 +48,52 @@ sidecars:
       DD_API_KEY: /copilot/chronos/dev/secrets/DATADOG_API_KEY
     variables:
       ECS_FARGATE: true
+      DD_LOGS_ENABLED: true
+      DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL: true
       DD_APM_ENABLED: true
-      DD_SERVICE: chronos-rails
       DD_DOCKER_ENV_AS_TAGS: true
+      DD_SERVICE: chronos-rails
 
 logging:
   destination:
-    # https://docs.aws.amazon.com/AmazonECS/latest/userguide/firelens-example-taskdefs.html#firelens-example-firehose
-    Name: firehose
-    region: ap-northeast-1
-    delivery_stream: chronos-rails-datadog
+    Name: datadog
+    Host: http-intake.logs.datadoghq.com
+    TLS: 'on'
+    dd_service: chronos-rails
+    dd_source: ruby
+    provider: ecs
+  enableMetadata: true
+  secretOptions:
+    apikey: /copilot/chronos/dev/secrets/DATADOG_API_KEY
+  # To parse Rails logs formatted as JSON by Lograge
+  # https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/parse-json
+  configFilePath: /fluent-bit/configs/parse-json.conf
```

Comment:
>  If your containers are publishing serialized JSON logs over stdout, you should use this extra firelens configuration to get them correctly parsed within Datadog:
```json
{
  "essential": true,
  "image": "amazon/aws-for-fluent-bit:latest",
  "name": "log_router",
  "firelensConfiguration": {
    "type": "fluentbit",
    "options": {
      "enable-ecs-log-metadata": "true",
      "config-file-type": "file",
      "config-file-value": "/fluent-bit/configs/parse-json.conf"
    }
  }
}
```
> This converts serialized JSON from the log: field into top-level fields. See the AWS sample Parsing container stdout logs that are serialized JSON for more details.
> ref: https://app.datadoghq.com/account/settings#integrations/aws-fargate

- [FireLens Example: Parsing container stdout logs that are serialized JSON
](https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/parse-json)